cves/CVE-2012-0858.yml

```diff
@@ -243,8 +248,8 @@ subsystem:
     If this involves fixing multiple subsystems, you can make this an array.
 
     In 'answer', explain how you arrived at this determination.
-  answer:
-  name:
+  answer: the shorten file related to this vulnerability is a part of avcodec
```

Comment:
An explanation of the main/major functionalities of the shorten file would help to better understand its purpose and why this is part of the avcodec subsystem.

Reply:
Described what the Shorten file is and what it does